CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -805,9 +808,21 @@ protected static function insertGroupContactsIntoTempTable(string $tempTableName
       $addSelect = "$groupID AS group_id";
 
       if (!empty($ssParams['api_entity'])) {
-        $sql = self::getApiSQL($ssParams, $addSelect, $excludeClause);
+        $sql = self::getApiSQL($savedSearch, $addSelect, $excludeClause);
       }
       else {
+        $fv = CRM_Contact_BAO_SavedSearch::getFormValues($savedSearchID);
```

Comment:
In fact I will push this into the following sql functions as a follow up - but it needs to be combined with the line below